src/components/SmartLoan.vue

```diff
@@ -0,0 +1,327 @@
+<template>
+  <div class="smart-loan container">
+    <Bar>
+      <div>
+        <Value label="Loan"
+          :primary="{value: debt, type: 'avax', showIcon: true}"
+          :secondary="{value: toUSD(debt), type: 'usd'}"
+          :flexDirection="isMobile ? 'row' : 'column'" />
+          <div class="borrow-buttons">
+            <img @click="showBorrowBlock(0)" src="src/assets/icons/plus.svg" class="plus"/>
+            <img src="src/assets/icons/slash-small.svg"/>
+            <img @click="showBorrowBlock(1)" src="src/assets/icons/minus.svg" class="minus"/>
+          </div>
+      </div>
+      <div class="solvency-value">
+        <div class="label">
+          Solvency
+        </div>
+        <div class="solvency-gauge">
+          <SolvencyBar />
+        </div>
+      </div>
+      <div>
+        <Value label="Collateral"
+          :primary="{value: collateral, type: 'avax', showIcon: true}"
+          :secondary="{value: toUSD(collateral), type: 'usd'}"
+          :flexDirection="isMobile ? 'row' : 'column'" />
+          <div class="fund-buttons">
+            <img @click="showCollateralBlock(0)" src="src/assets/icons/plus.svg" class="plus"/>
+            <img src="src/assets/icons/slash-small.svg"/>
+            <img @click="showCollateralBlock(1)" src="src/assets/icons/minus.svg" class="minus"/>
+          </div>
+      </div>
+    </Bar>
+    <Block v-if="borrowBlock" class="block borrow-block" :bordered="true">
+      <img @click="borrowBlock = false" src="src/assets/icons/cross.svg" class="cross" />
+      <Tabs :openTabIndex="tabIndex">
+        <Tab title="Borrow" imgActive="add-deposit-active" img="add-deposit" imgPosition="left">
+          <CurrencyForm
+            label="Borrow"
+            v-on:submitValue="borrowValue"
+            :waiting="waitingForBorrow"
+            flexDirection="column"
+            :style="{'width': '490px'}"
+            :validators="[
+              {require: value => value <= getAvailable, message: 'Borrow amount exceeds amount available in the pool'},
+              {require: value => calculateSolvency(0, value) >= 1.2, message: 'New solvency ratio is below acceptable level'},
+            ]"
+            :info="value => `New solvency ratio: <b>${(calculateSolvency(0, value) * 100).toFixed(1)}%</b>`"
+          />
+        </Tab>
+        <Tab title="Repay" imgActive="withdraw-deposit-active" img="withdraw-deposit" imgPosition="right">
+          <CurrencyForm
+            label="Repay"
+            v-on:submitValue="repayValue"
+            :waiting="waitingForRepay"
+            flexDirection="column"
+            :style="{'width': '490px'}"
+            :validators="[
```

Comment:
I would move the :validators and :info values to the script part of the component